apps/coscrad-frontend/src/components/resources/photographs/photograph-index.presenter.tsx

```diff
@@ -1,13 +1,50 @@
 import { IIndexQueryResult, IPhotographViewModel } from '@coscrad/api-interfaces';
-import { GenericIndexPresenter } from '../../../utils/generic-components/presenters/generic-index-presenter';
+import { Link } from 'react-router-dom';
+import {
+    GenericIndexTablePresenter,
+    HeadingLabel,
+    Renderer,
+} from '../../../utils/generic-components/presenters/tables/generic-index-table-presenter';
 
-export const PhotographIndexPresenter = (
-    photographIndexResult: IIndexQueryResult<IPhotographViewModel>
-) => {
+export const PhotographIndexPresenter = (indexResult: IIndexQueryResult<IPhotographViewModel>) => {
     /**
      *  TODO [https://www.pivotaltracker.com/story/show/183681839]
      * We may some day read the actions and allow for bulk command execution in
      * an index view.
      */
-    return <GenericIndexPresenter {...photographIndexResult} />;
+    const { data: detailResult } = indexResult;
+
+    const photographs = detailResult.map(({ data }) => data);
+
+    const headingLabels: HeadingLabel<IPhotographViewModel>[] = [
+        { propertyKey: 'id', headingLabel: 'Link' },
+        { propertyKey: 'imageURL', headingLabel: 'Image URL' },
+        { propertyKey: 'photographer', headingLabel: 'Photographer' },
+    ];
+
+    /**
+     * Note that for some reason we don't get type safety unless we use the
+     * generics on the Map. We might want an abstraction that takes in an array
+     * or object data structure (in a type safe way) and creates this map.
+     */
+    const cellRenderers = new Map<keyof IPhotographViewModel, Renderer<IPhotographViewModel>>()
```

Comment:
We should consider passing in a type-safe object or array and allowing the util to new up the Map.

Reply:
done